content/news/XXXX-XX-XX-mixxx-2_5-released.md

```diff
@@ -0,0 +1,564 @@
+* Skins: Fix checkbox styling on Qt 6 [#12050](https://github.com/mixxxdj/mixxx/pull/12050)
+* Skins: Fix Tango waveform splitter [#12939](https://github.com/mixxxdj/mixxx/pull/12939)
+* Skin: Fix Tango rate range label position [#13242](https://github.com/mixxxdj/mixxx/pull/13242)
+* Introduce wrapper for non const iterators for erase and insert
+  [#12201](https://github.com/mixxxdj/mixxx/pull/12201)
+  [#13856](https://github.com/mixxxdj/mixxx/pull/13856)
+* Fix Qt6/QML build [#12255](https://github.com/mixxxdj/mixxx/pull/12255)
+* Fix track color background with Qt6 [#12380](https://github.com/mixxxdj/mixxx/pull/12380)
+* multi-line delegate: fix bg color, Qt6 on Linux
+  [#12478](https://github.com/mixxxdj/mixxx/pull/12478)
+* Revert "BaseTrackPlayer: Remove references to WaveformWidgetRenderer when using Qt6" [#12342](https://github.com/mixxxdj/mixxx/pull/12342)
+* Fix: Replace deprecated `qAsConst` with `std::as_const` [#13028](https://github.com/mixxxdj/mixxx/pull/13028)
+* Fix Drag'n'drop: avoid unintended drag on hover (WTrackProperty, WCoverArt etc.)
+  [#13035](https://github.com/mixxxdj/mixxx/pull/13035)
+  [#13033](https://github.com/mixxxdj/mixxx/issues/13033)
+* Fix ambiguous overload error due to native qDebug impl for std::optional
+  [#12981](https://github.com/mixxxdj/mixxx/issues/12981)
+* Workaround for Qt6 'selected click' bug [#12488](https://github.com/mixxxdj/mixxx/pull/12488)
+* Fix menu icon position [#12216](https://github.com/mixxxdj/mixxx/pull/12216)
+* Fix/Skins: hide reserved space for invisible table header sort indicators [#13535](https://github.com/mixxxdj/mixxx/pull/13535)
+* Qt 6.8 deprecated declaration fixes [#13845](https://github.com/mixxxdj/mixxx/pull/13845)
+* Add missing qt6-declarative-private-dev and qt6-base-private-dev package [#13904](https://github.com/mixxxdj/mixxx/pull/13904)
+
+### Experimental iOS support
+
+* CMakeLists: Support building for iOS [#12672](https://github.com/mixxxdj/mixxx/pull/12672)
+* DlgPrefInterface: Disable tooltips on iOS by default [#12689](https://github.com/mixxxdj/mixxx/pull/12689)
+* SoundManager: Set up `AVAudioSession` on iOS [#12714](https://github.com/mixxxdj/mixxx/pull/12714)
+* SoundManager: Use correct PortAudio backend on iOS [#12716](https://github.com/mixxxdj/mixxx/pull/12716)
+* DesktopHelper: Add openUrl abstraction to support iOS [#12698](https://github.com/mixxxdj/mixxx/pull/12698)
+* iOS packaging: Add Info.plist, launch screen and app icon [#12676](https://github.com/mixxxdj/mixxx/pull/12676)
+* CmdlineArgs: Move config directory to a user-accessible location on iOS
+  [#12688](https://github.com/mixxxdj/mixxx/pull/12688)
+* CMakeLists: Work around Qt shader bug with Xcode
+  [#13379](https://github.com/mixxxdj/mixxx/pull/13379)
+  [#13378](https://github.com/mixxxdj/mixxx/issues/13378)
+* AudioUnitManager: Disable unavailable in-process instantiation on iOS
+  [#13383](https://github.com/mixxxdj/mixxx/pull/13383)

```

Comment:
```suggestion
```
iOS support is not available for end users in this release

Reply:
thank you, that will make the list even more shorter, readers will still be able to find everything in the complete changelog.